src/js/schemas/service-schema/General.js

```diff
@@ -22,29 +22,33 @@ let General = {
           title: 'CPUs',
           description: 'The amount of CPUs which are used for the service',
           type: 'number',
+          default: 1,
           getter: function (service) {
             return `${service.getCpus() || ''}`;
           }
         },
         mem: {
           title: 'Mem (MiB)',
           type: 'number',
+          default: 128,
           getter: function (service) {
             return `${service.getMem() || ''}`;
           }
         },
         disk: {
           title: 'Disk (MiB)',
           type: 'number',
+          default: 0,
           getter: function (service) {
-            return `${service.getDisk() || ''}`;
+            return `${service.getDisk() || 0}`;
           }
         },
         instances: {
           title: 'Instances',
           type: 'number',
+          default: 1,
           getter: function (service) {
-            return `${service.getInstancesCount() || 0}`;
+            return `${service.getInstancesCount() || ''}`;
```

Comment:
but right here, you're still return `''` not `0`, no?

if you change this feel free to merge